### PR TITLE
Fix 32bit scale

### PR DESCRIPTION
**Pull request by lukamac**

### CODE REVIEW

File: ne16/README.md
Comment (FrancescoConti) [inline ne16/README.md:1]: I would remove `uint16` actually... even if it's there in HW

**Review by FrancescoConti — APPROVED** [review]
LGTM

File: ne16/README.md
Comment (lukamac) [reply ne16/README.md:1]: Like completely remove `mode16`?

Comment (FrancescoConti) [reply ne16/README.md:1]: I refer specifically to the scaling option. Not to the full support for 16-bits inputs.

**Pull request merged**